dwh/using_fhir_views.ipynb

```diff
@@ -0,0 +1,814 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to using FHIR Data Pipes and FHIR Views\n",
+    "In this example we will show how to use FHIR Views with data generated using the FHIR Data Pipes."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prerequisites:\n",
+    "\n",
+    "**Note**: All commands need to be run from the root directory of the fhir-data-pipes repo\n",
+    "\n",
+    "1. Bring up a Hive ThriftServer:\n",
+    "    ```\n",
+    "    docker-compose  -f ./docker/compose-controller-spark-sql-single.yaml up --build  --force-recreate -d\n",
+    "    ```\n",
+    "\n",
+    "2. Bring up a HAPI FHIR server:\n",
+    "    ```\n",
+    "    docker-compose  -f ./docker/hapi-compose.yml up --force-recreate -d\n",
+    "    ```\n",
+    "\n",
+    "3. Load data into the HAPI FHIR server:\n",
+    "    ```\n",
+    "    python3 ./synthea-hiv/uploader/main.py HAPI http://localhost:8091/fhir --input_dir ./synthea-hiv/sample_data\n",
+    "    ```\n",
+    "\n",
+    "4. In the FHIR Pipelines Control Panel page, http://localhost:8090, click on “Run Full” button to convert the data to Parquet files with SQL-on-FHIR schema. The exported Parquet files will be stored under the `docker/dwh` directory.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Installation\n",
+    "\n",
+    "It's recommended to set up a virtual environment before starting the notebook kernel to avoid any dependency version issues with your native environment.\n",
+    "\n",
+    "This can be done with the following commands in a terminal, if you have Conda installed:\n",
```

Comment:
It is probably better to make this independent of Conda; it is a minor issue we can leave for future.